ONSdigital/eq-author-app
# EAR 2192 additional content on list collector page

Ticket: https://jira.ons.gov.uk/browse/EAR-2192

### What is the context of this PR?

> Add additional guidance to the collection question by adding the additional content collapsible on the add item page in the list collector folder

### How to review
1. Create a collection list
2. Add a list collector
3. In the add item page, check the following:

> - [ ] The additional content collapsible follows the Figma [prototype](https://www.figma.com/file/RmplTFLXmJGKQ1VnDk5uRo/EAR---2192-Provide-additional-content-on-the-collection-question?type=design&node-id=0-1&mode=design)
> - [ ] The toggle switches work as they should when toggled on/off
> - [ ] Adding text input into the additional content fields works as they should
> - [ ] The correct error messages are displayed when there is no content 
> - [ ] The preview page shows the additional content fields correctly and follows the same design as the one on the question page preview
> - [ ] Check the other list collector pages do not have the additional content collapsible.
> - [ ] Check the additional content collapsible displays when duplicating a list collector.
> - [ ] Check the migration works for the additional content collapsible using an old questionnaire with a list collector
